### ISSUE TITLE
Missing alt text on primary image

### ISSUE THREAD

**Issue by brandonhundt** [PRX/theworld.org#194]
The main image on the standard template is missing alt text.

1. Go to this story: https://preview.theworld.org/stories/2015-12-14/lessons-new-zealand-s-disappointing-and-now-complete-flag-referendum
2. Inspect the main image and note alt text is missing.
3. Alt text is provided on the Drupal version: www.pri.org/stories/2015-12-14/lessons-new-zealand-s-disappointing-and-now-complete-flag-referendum

(also double check featured image on story pages too.)

**Comment by rpeterman-gp** [comment]
Already caught this in metatags work.
